client/src/app.css

```
@@ -44,3 +44,7 @@ a {
 #active {
 	background-color: #b26910;
 }
+
+.projectCardLinks {
+	@apply border border-white border-solid w-10 h-10 rounded-2xl flex justify-center items-center hover:border-projectCardLinkHover hover:text-projectCardLinkHover after:content-[attr(data-tooltip)] after:absolute after:-translate-y-full after:bottom-3 after:bg-projectCardLinkHover after:text-projectCardBg after:p-1 after:rounded after:hidden hover:after:block  before:border-8 before:border-transparent before:border-solid before:border-t-projectCardLinkHover before:absolute before:-translate-y-full before:bottom-4 before:hidden hover:before:block before:transition-all after:transition-all;
+}
```

Please create `project.css` file in `./component/project/*` and add your tailwind function directive class. It is much easier to trace back the project style. 

Use the hyphen class named tailwind to make it more consistent. The icon looks big. I recommend adding p-1.5 to shrink the icon size. If you have any other methods, feel free to use it.

```suggestion
.project-card-link {
	@apply border border-white border-solid w-10 h-10 rounded-full p-1.5 flex justify-center items-center hover:border-projectCardLinkHover hover:text-projectCardLinkHover after:content-[attr(data-tooltip)] after:absolute after:-translate-y-full after:bottom-3 after:bg-projectCardLinkHover after:text-projectCardBg after:p-1 after:rounded after:hidden hover:after:block  before:border-8 before:border-transparent before:border-solid before:border-t-projectCardLinkHover before:absolute before:-translate-y-full before:bottom-4 before:hidden hover:before:block before:transition-all after:transition-all;
}
```